instant-xml-macros/src/meta/mod.rs

```diff
@@ -0,0 +1,14 @@
+mod container;
```

Comment:
IMO this is still overindexing on small modules. I think it would make sense to move `meta.rs` to `meta/mod.rs`, and only split out `namespace` into a separate file for now.